Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -1308,43 +1305,6 @@ theorem DeleteFar.mono (h : G.DeleteFar p r₂) (hr : r₁ ≤ r₂) : G.DeleteF
 
 end DeleteFar
 
-/-! ## Vertex replacement -/
```

Comment:
What's the motivation for moving this?

Reply:
I feel that `Combinatorics.SimpleGraph.Basic` is already too long (2000+ lines), and that the edge cardinality theorems I've proven about the local operations are not basic at all. (Said theorems also total 150+ lines.)

Reply:
Can you split this to a standalone PR, making clear that it's moved without modification?

Reply:
#9400